{
	"name": "CVE-2020-1738",
	"description": "A flaw was found in Ansible Engine when the module package or service is used and the parameter 'use' is not specified. If a previous task is executed with a malicious user, the module sent can be selected by the attacker using the ansible facts file. All versions in 2.7.x, 2.8.x and 2.9.x branches are believed to be vulnerable.",
	"packages": [
		{
			"name": "ansible",
			"releases": {
				"bookworm": {
					"status": "open",
					"repositories": {
						"bookworm": "5.5.0-1"
					},
					"urgency": "unimportant"
				},
				"bullseye": {
					"status": "open",
					"repositories": {
						"bullseye": "2.10.7+merged+base+2.10.8+dfsg-1"
					},
					"urgency": "unimportant"
				},
				"buster": {
					"status": "open",
					"repositories": {
						"buster": "2.7.7+dfsg-1+deb10u1",
						"buster-security": "2.7.7+dfsg-1+deb10u1"
					},
					"urgency": "unimportant"
				},
				"sid": {
					"status": "open",
					"repositories": {
						"sid": "5.5.0-1"
					},
					"urgency": "unimportant"
				},
				"stretch": {
					"status": "open",
					"repositories": {
						"stretch": "2.2.1.0-2+deb9u1",
						"stretch-security": "2.2.1.0-2+deb9u3"
					},
					"urgency": "unimportant"
				}
			}
		}
	],
	"scope": "local"
}